test_settings.py

```diff
@@ -65,4 +65,17 @@ def root(*args):
 FORUM_PORT = "4567"
 FORUM_MONGO_HOST = "mongo-test-url"
 FORUM_MONGO_PORT = 27017
+
 FORUM_ENABLE_ELASTIC_SEARCH = False
+FORUM_ELASTICSEARCH_INDEX_NAMES = ["comment_threads", "comments"]
+FORUM_MAX_DEEP_SEARCH_COMMENT_COUNT = 1000
+ELASTIC_SEARCH_CONFIG = [
+    {
+        "host": "localhost",
+        "port": "5200",
+        "use_ssl": False,
+    }
+]
+
+FORUM_DEFAULT_PAGE = 1
+FORUM_DEFAULT_PER_PAGE = 20
```

Comment:
I believe it's best to store these values in the settings since we use them across multiple files. I've noticed that the user API also relies on these values.

Reply:
If we reuse these values across multiple files, but they are not modified from one platform to the next, then we should move them to a different module.